### ISSUE TITLE
支持支持！

### ISSUE THREAD

**Issue by sunanlin13174** [Code-keys/YOLOv5-AF#1]
催更。。。

**Comment by Zephyr69** [comment]
I'm expecting this too. An anchor-free yolov5!

**Comment by Code-keys** [comment]
How about YOLOX.

In my opinion, the value of this project is ha-ha.

**Comment by Zephyr69** [comment]
> How about YOLOX.
> 
> In my opinion, the value of this project is ha-ha.

Yeah I closely monitor both projects. But yolov5 repo currently has way cleaner and more flexible customization routes, which is heavily required in my works.
